hedera-node/configuration/mainnet/upgrade/feeSchedules.json

```diff
@@ -1865,38 +1865,38 @@
             {
               "subType": "DEFAULT",
               "nodedata": {
-                "constant": 4574479485,
-                "bpt": 7313222,
+                "constant": 0,
+                "bpt": 100000000000,
```

Comment:
I cannot read these numbers. Doesn't JSON support scientific notation? That would make it much easier to read the values.

Reply:
Yes, I think it supports. We will need to change all values in this file I think. Will do in a separate PR. Thanks @netopyr